src/scenario.py

```diff
@@ -188,7 +193,192 @@ def run(self, strategy_name, options):
+                count_window[widx] = list(map(
+                    lambda c, t: c + (t is not None),
+                    count_window[widx], socs[idx]))
+
+                for i, soc in enumerate(socs[idx]):
+                    if soc is None and load_count[i][-1] > 0:
+                        load_count[i].append(0)
+                    else:
+                        load_count[i][-1] += (soc is not None)
+
+                fixed_load = sum([v for k, v in extLoads[idx].items() if
+                                  k in self.events.external_load_lists or
+                                  k in self.events.energy_feed_in_lists])
+                max_fixed_load = max(max_fixed_load, fixed_load)
+                var_load = totalLoad[idx] - fixed_load
+                max_variable_load = max(max_variable_load, var_load)
+
+            ts_per_hour = datetime.timedelta(hours=1) / self.interval
+
+            # avg flex per window
+            avg_flex_per_window = [sum([t[0] for t in w]) / len(w) if w else 0 for w in load_window]
+            json_results["avg flex per window"] = {
+                "04-10": avg_flex_per_window[0],
+                "10-16": avg_flex_per_window[1],
+                "16-22": avg_flex_per_window[2],
+                "22-04": avg_flex_per_window[3],
+                "unit": "kW",
+                "info": "Average Flex-power range per time window"
+            }
+
+            # sum of used energy per window
+            sum_energy_per_window = [sum([t[1] for t in w]) / ts_per_hour for w in load_window]
+            json_results["sum of energy per window"] = {
+                "04-10": sum_energy_per_window[0],
+                "10-16": sum_energy_per_window[1],
+                "16-22": sum_energy_per_window[2],
+                "22-04": sum_energy_per_window[3],
+                "unit": "kWh",
+                "info": "Total drawn energy per time window"
+            }
+
+            # avg standing time
+            # don't use info from flex band, as standing times might be interleaved
+            # remove last empty standing count
+            for counts in load_count:
+                if counts[-1] == 0:
+                    counts = counts[:-1]
+            num_loads = sum(map(len, load_count))
+            if num_loads > 0:
+                avg_stand_time = sum(map(sum, load_count)) / ts_per_hour / num_loads
+            else:
+                avg_stand_time = 0
+            # avg total standing time
+            # count per car: list(zip(*count_window))
+            total_standing = sum(map(sum, count_window))
+            avg_total_standing_time = total_standing / len(self.constants.vehicles) / ts_per_hour
+            json_results["avg standing time"] = {
+                "single": avg_stand_time,
+                "total": avg_total_standing_time,
+                "unit": "h",
+                "info": "Average duration of a single charge event and total time spent charging"
```

Comment:
The info shoud be: "Average duration of a single standing event and average total time standing time of all vehicles"
It seems to be calculated correct though. Or does "load_count" calculate the charging time?